llvmlite/binding/ffi.py

```diff
@@ -30,6 +31,82 @@ def _make_opaque_ref(name):
 LLVMSectionIteratorRef = _make_opaque_ref("LLVMSectionIterator")
 
 
+class _lib_wrapper(object):
+    """Wrap the libllvmlite with a lock such that only one thread is
+    accessing it at a time.
```

Comment:
Perhaps `Wrap libllvmlite with a lock such that only one thread may access it at a time`?